chains/molten/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://molten.calderaexplorer.xyz/api
+    family: blockscout
+    name: Caldera Molten Explorer
+    url: https://molten.calderaexplorer.xyz
+blocks:
+  confirmations: 1
+  estimateBlockTime: 1
+  reorgPeriod: 0
+chainId: 360
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
```

Comment:
This looks incorrect. You should set your information here.

Reply:
its my understanding that someone from hyperlane would assist with the deployment of the contracts using the new tooling. And so the contract deployers would be hyperlane itself rather than us (molten).